tests/system/robot/symbolPronunciationTests.py

```diff
@@ -195,14 +195,14 @@ def test_moveByLine():
 		symbolLevel=SymLevel.NONE,
 		expectedSpeech=[
 			'Say', '(quietly)', 'Hello,', 'Jim .', "don't",  # Expect:
-			'',  # todo: Expect 'right-pointing arrow'
```

Comment:
Don't remove these expected symbol todo comments. These tests were added when assessing the current behavior for symbols. The todo comments document what we intend the output to become. 

So while "blank" is better than `""`, it still isn't the final goal.

Reply:
> Don't remove these expected symbol todo comments. These tests were added when assessing the current behavior for symbols. The todo comments document what we intend the output to become.
> 
> So while "blank" is better than `""`, it still isn't the final goal.

This was actually on purpose.  I wanted to spark a discussion about this. 

I would propose that this perhaps should be the final goal. If symbol level is none, then as a user I would expect no symbols would be spoken. If a speech sequence doesn't contain anything to be spoken, "blank" should be spoken instead. 

Otherwise, I don't understand why "'right-pointing arrow" should be spoken when symbol level is none. Shouldn't it be considered a symbol and so not spoken? If this was discussed and this is indeed the intended final goal then at least for me it is confusing and I think the test case would benefit from having a comment with a small explanation about the justification

Reply:
Blank is misleading. It suggests there are no visible characters in the range. We've had quite some discussions about this already. The should be given the most accurate picture as a priority. This might mean announcing the name of a single symbol, or "symbols" when there are multiple, perhaps like spaces / tabs the number should be spoken "5 symbols". A mix of whitespace should perhaps be announced as "mixed whitespace".

Our last work in this area was really focusing on symbols and the inconsistency of reporting.

Reply:
hmm, okay. This reasoning was missing in those tests making it unclear why that was the expected output. I will add back those todos along with short comment on the why, as you just explained it, so others are not confused as I was.

However, what about the case of a sequence that only contains tabs? In that case isn't "blank" an appropriate final solution, since tabs are whitespace and so would generally be considered as blank?